### PR TITLE
Added Unix Support

### PR DESCRIPTION
**Pull request by wyatt-herkamp**
This adds Unix support to the Time editor. Unix does not store creation time.

### CODE REVIEW

File: src/metadata/time.rs [hunk 1]
```diff
@@ -48,9 +51,9 @@ impl FileTime {
             day,
             month,
             year,
-            hour: -1,
-            minute: -1,
-            second: -1,
+            hour: 0,
```
Comment (ryandw11) [inline src/metadata/time.rs]: These need to be -1 as -1 denotes to use the current system time.

**Review by ryandw11 — CHANGES_REQUESTED** [review]
Looks good for the most part.

File: src/metadata/time.rs (hunk 1 above)
Comment (wyatt-herkamp) [reply src/metadata/time.rs]: Fixed

**Review by ryandw11 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**